helpers.py

```diff
@@ -24,3 +24,47 @@ def sendVerificationEmail(mail, to, first_name, last_name, verification_token):
         return None
     except Exception as e:
         return e
+
+def sendApprovedEmail(mail, to, first_name, last_name):
+    # Create the callback URL
+    callback = ''
+    if app.config["ENV"] == 'development':
+        callback = os.getenv("FRONTEND_URL")
+    else:
+        callback = os.getenv("PROD_ROOT_URL")
```

Comment:
```suggestion
        callback = os.getenv("DEV_FRONTEND_URL")
```
The callback in the approval email should be the frontend url, not backend